src/services/completions.ts

```diff
@@ -3385,4 +3390,18 @@ namespace ts.Completions {
         }
         return charCode;
     }
+
+    function isTypeOnlyImportDeclaration(node: Node) {
+        switch (node.kind) {
+            case SyntaxKind.ImportSpecifier:
+                return (node as ImportOrExportSpecifier).parent.parent.isTypeOnly;
+            case SyntaxKind.ImportClause:
+            case SyntaxKind.ImportEqualsDeclaration:
+                return (node as ImportClause | ImportEqualsDeclaration).isTypeOnly;
+            case SyntaxKind.NamespaceImport:
+                return (node as NamespaceImport).parent.isTypeOnly;
+            default:
+                return false;
+        }
+    }

```

Comment:
You can just use `isTypeOnlyImportOrExportDeclaration` already defined in utilities since `importCompletionNode` will be undefined if you’re actually on an export specifier.

Reply:
Oke, thanks